src/components/molecules/CuriosityFacts.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+const CurosityFacts = () => {
```

Comment:
Same thing of the props
```suggestion
const CurosityFacts = ({ title, description }) => {
```

Reply:
Ready